samples/nodejs_simple_sample/README.md

```diff
@@ -7,35 +7,39 @@ This document describes how to prepare your development environment to use the *
 
 ## Prerequisites
 - Install [Python 2.7](https://www.python.org/downloads/release/python-2712/)
-- Clone https://github.com/Azure/azure-iot-gateway-sdk.git
+- [Prepare your development environment](../../doc/devbox_setup.md)
 
 ## Building
 ### Windows
-Open a Developer Command Prompt for VS and: 
-
-- cd <\<your azure repo>>\tools
-- Run build_nodejs.cmd ==> build_nodejs.cmd downloads and builds Node JS from source as runtime linked modules.
-- Set 2 environment variables : `NODE_INCLUDE=<your azure-iot-gateway_root>\build_nodejs\dist\inc` and `NODE_LIB=<your aziore-iot-gateway_root>\%build-root%\build_nodejs\dist\lib`. These envioronment variables are a path to node.dll library and header files used by our Node JS binding. 
-- run build.cmd --enable-nodejs-binding
+From a Visual Studio Developer Command Prompt:
+- `cd <azure_iot_gateway_sdk_root>\tools`
+- `set PATH=%PATH%;C:\Python27`
```

Comment:
This step isn't usually necessary...newer versions of Python 2.7 update the PATH by default. Can you remove this bullet from here, and add it under the "Install Python 2.7" rerequisite above? Say something like, "Make sure Python is added to your PATH."